components/CommentFrom.jsx

```diff
@@ -0,0 +1,88 @@
+'use client'
+import React, { useState } from "react";
+
+const CommentForm = () => {
+  const [formData, setFormData] = useState({
+    name: "",
+    comment: ""
+  });
+
+  const handleChange = (e) => {
+    setFormData({
+      ...formData,
+      [e.target.name]: e.target.value,
+    });
+  };
+  const [isSubmitting, setIsSubmitting] = useState(false);
+  const handleSubmit = async (e) => {
+    e.preventDefault(); // Prevent default form submission
+    setIsSubmitting(true); // Set submitting state
+    try {
+      const response = await fetch("http://localhost:8000/api/v1/comment/:id", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(formData), // Send the form data as JSON
+      });
+
+      if (!response.ok) {
+        throw new Error("Network response was not ok");
+      }
+
+      const data = await response.json();
+      console.log("Success:", data);
+      // Optionally reset the form or show success message
+    } catch (error) {
+      console.error("Error:", error);
+    } finally {
+      setFormData({ name: "", comment: "" });
+      setIsSubmitting(false); // Reset submitting state
+    }
+  };
```

Comment:
@sagarverma2602 and @RameshPolubothu 
Please pick this up today. It's always the best if we have configured API endpoints rather than hard coded ones.